requirements.txt

```diff
@@ -1,3 +1,11 @@
-SimpleITK
-vtk
-numpy
+astroid==2.4.1
```

Comment:
My conda env doesn't have astroid, isort, lazy-object-proxy, mccabe, pylint, toml, or wrapt.

Reply:
I'm going to merge the pull request and then edit the requirements.txt file afterwards.

Reply:
Hi Dave
Tracked down those dependencies, they came from pylint 

```
pylint==2.5.2
  - astroid [required: >=2.4.0,<=2.5, installed: 2.4.1]
    - lazy-object-proxy [required: ==1.4.*, installed: 1.4.3]
    - six [required: ~=1.12, installed: 1.15.0]
    - wrapt [required: ~=1.11, installed: 1.12.1]
  - isort [required: >=4.2.5,<5, installed: 4.3.21]
  - mccabe [required: >=0.6,<0.7, installed: 0.6.1]
  - toml [required: >=0.7.1, installed: 0.10.1]
setuptools==46.4.0
SimpleITK==1.2.4
vtk==9.0.0
wheel==0.34.2
```

You should be good to go without them. 
Thanks
Patrick